lessons/module-3/react-2-the-how.md

```diff
@@ -185,62 +175,55 @@ Let's figure out what should be a component in our app.
 
 Because we want App to hold onto our list of ideas, let's think about how we're going to store that information.
 
-In our first React lesson, we learned about _state_. In order to use component state, we need our component to be a class.
+In our first React lesson, we learned about _state_. In order to use component state, we need to import useState from React.
 
-So: let's import React and the parent Component class, and create our child App component! Think back to your OOP knowhow from Mod 2.
+So: let's import useState from react, and create our child App component!
 
 ```js
-import React, { Component } from 'react';
+import { useState } from 'react';
+function App() {
+ 
+ // Any JSX code or additional logic will go here
 
-class App extends Component {
-  constructor() {
-    super();
-  }
+ return (
+  //Your JSX code goes here
+ )
 }
 
 export default App;
 ```
 
-When we save that, our error now says "TypeError: instance.render is not a function". Why are we getting this error?
-
-The class method `render` comes from the parent class, Component.
+The above code is defining the App functional component in React. 
 
-Just like any other parent class in OOP, when we inherit from it to create a child class, we have access to all of the parent's methods and properties. This is why the first thing we have to put in our constructor is the familiar `super` function.
+Any additional JSX code or logic can be added to the component as needed, and the final JSX code is returned by the component's return statement just like a pure function.
 
-If you look under the hood, the React "Component" is just a class that contains various methods (`render`, `setState`, `componentDidMount`). We will learn more about these in this and the next lesson.
+It's important to note that this code is using functional components, rather than class components and OOP. This allows us to manage our component in a more concise and streamlined way, without relying on class inheritance and traditional OOP principles.
 
 Let's keep writing our App component!
 
 ```js
-import React, { Component } from 'react';
-
-class App extends Component {
-  constructor() {
-    super();
-  }
-
-  render() {
+import  {useState} from 'react';
```

Comment:
```import { useState } from 'react';```

We lost the spaces around the destructured `useState` - I think we should keep them to keep things consistent!